src/controllers/session_controller.ts

```diff
@@ -12,34 +12,33 @@ import { onNewLogin } from '../services/new_login'
 import logger from '../services/logger'
 
 export class SessionController {
-
   private getConfig: getConfig
   private getClient: getClient
   private incoming: Incoming
   private outgoing: Outgoing
 
   constructor(incoming: Incoming, outgoing: Outgoing, getConfig: getConfig, getClient: getClient) {
     this.incoming = incoming
-    this.outgoing = outgoing    
+    this.outgoing = outgoing
     this.getConfig = getConfig
     this.getClient = getClient
-  }  
+  }
 
   public async info(req: Request, res: Response) {
     logger.debug('info method', req.method)
     logger.debug('info headers', req.headers)
     logger.debug('info params', req.params)
     logger.debug('info body', JSON.stringify(req.body, null, ' '))
-    logger.debug('info query', JSON.stringify(req.query, null, ' '))    
+    logger.debug('info query', JSON.stringify(req.query, null, ' '))
     const { phone } = req.params
     try {
       if (clients && clients.has(phone)) {
```

Comment:
aqui não poderia usar o `this.getClient(phone)` no lugar de `clients && clients.has(phone)`?

Reply:
@giquieu ?